google/cloud/spanner/client_test.cc

```diff
@@ -981,6 +981,28 @@ TEST(ClientTest, CommitStats) {
   EXPECT_EQ(42, result->commit_stats->mutation_count);
 }
 
+TEST(ClientTest, MaxCommitDelay) {
+  auto timestamp =
+      spanner_internal::TimestampFromRFC3339("2020-10-20T02:20:09.123Z");
+  ASSERT_STATUS_OK(timestamp);
+  CommitStats stats{42};
```

Comment:
Please remove the `CommitStats` from this test.  We neither request or expect them.